angles/src/angles/__init__.py

```diff
@@ -0,0 +1,179 @@
+#*********************************************************************
+# Software License Agreement (BSD License)
+#
+#  Copyright (c) 2008, Willow Garage, Inc.
```

Comment:
You wrote the new code. You get the copyright (though if you're working for a company you may be required to give it to them.). And it should be dated the year written. 

When you significantly edit a file, you should add the name and year at the top to a list in the header.